akka-http-core/src/test/scala/akka/http/javadsl/model/headers/HttpCookieSpec.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.http.javadsl.model.headers
+
+import java.util.{ Optional, OptionalLong }
+
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.must.Matchers
+
+class HttpCookieSpec extends AnyFreeSpec with Matchers {
```

Comment:
Makes sense to create those here. Test coverage of Java API is not too great so far.